src/Telegram.Bot/ITelegramBotClient.cs

```diff
@@ -474,10 +474,11 @@ public interface ITelegramBotClient
         /// </summary>
         /// <param name="chatId"><see cref="ChatId"/> for the target group</param>
         /// <param name="userId">Unique identifier of the target user</param>
+        /// <param name="untilDate"><see cref="DateTime"/> when the user will be unbanned. If user is banned for more than 366 days or less than 30 seconds from the current time they are considered to be banned forever</param>
         /// <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
         /// <returns><c>true</c> on success.</returns>
         /// <see href="https://core.telegram.org/bots/api#kickchatmember"/>
-        Task<bool> KickChatMemberAsync(ChatId chatId, int userId,
+        Task<bool> KickChatMemberAsync(ChatId chatId, int userId, DateTime untilDate = default(DateTime),
```

Comment:
I'm not really sure if this default value for `untilDate` is a good idea. I should be able to pass the following values and kick the user for ever:
```
DateTime.MinValue
DateTime.MaxValue
DateTime.UtcNow.AddSeconds(29)
DateTime.UtcNow.AddDays(366)
```

Also, is this properly parsed into UTC date time? Epoch time is based on UTC.

Here is how I resolved it: <https://github.com/TelegramBots/Telegram.Bot.Core/blob/master/src/Telegram.Bot/TelegramBotClient.cs#L848-L861>

And this is the test: <https://github.com/TelegramBots/Telegram.Bot.Core/blob/master/test/Telegram.Bot.Tests.Integ/AdminBots/ChatMemberAdministrationTests.cs#L139-L150>